README.md

```diff
@@ -2,147 +2,31 @@
 
 This project contains rke2 cluster template helm chart, which can be applied with values.yaml as configurations to create clusters.
 
-### How to use
+### Adding the Helm Chart via CLI
 
-The general cluster configuration options are available through [values.yaml](./charts/values.yaml).
+```bash
+helm repo add cluster-templates https://raw.githubusercontent.com/rancher/cluster-template-examples/main
+helm repo update
+```
+
+### Adding the Helm Chart via Rancher Manager
+
+1. Authenticate into the Rancher Manager UI.
+2. Navigate to **Cluster Management -> local** and click **Explore**.
```

Comment:
Don't have to go via "Cluster Management" specifically. I would change it to something more generic like below.
```suggestion
2. Open `local` cluster in **Cluster Explorer**
```